extensions/s3/src/main/java/io/deephaven/extensions/s3/BufferPool.java

```diff
@@ -1,9 +1,7 @@
 package io.deephaven.extensions.s3;
 
 import io.deephaven.base.reference.PooledObjectReference;
-import io.deephaven.base.reference.SimpleReference;
```

Comment:
Should I remove this change if we plan to back port this fix, to avoid merge conflicts?